administrator/language/en-GB/lib_joomla.ini

```
@@ -668,6 +668,7 @@ JLIB_INSTALLER_TUF_INVALID_METADATA="The saved TUF update information is invalid
 JLIB_INSTALLER_TUF_NOT_AVAILABLE="TUF is not available for extensions yet."
 JLIB_INSTALLER_TUF_ROLLBACK_ATTACK="Update not possible because the offered update is older than the currently installed version."
 JLIB_INSTALLER_TUF_SIGNATURE_THRESHOLD="Update not possible because the offered update does not have enough signatures."
+JLIB_INSTALLER_TUF_ERROR_GENERIC="Could not fetch update information, enable system debug mode for further information"
```

```suggestion
JLIB_INSTALLER_TUF_ERROR_GENERIC="Could not fetch update information, enable system debug mode for further information."
```

add period at the end of the sentence